.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "external/SharpZipLib"]
```

Comment:
oh, yes ... sorry, I just started this branch based off the other branch so I can start working on this feature. Once it's merged into master, these changes will no longer show up ... and if I change it based on your review/feedback, I'd simply rebase and `--force` push this branch.